util/src/main/i18n/templates/moderation.properties

```diff
@@ -150,6 +150,10 @@ moderation.defuse.enemy = You may not defuse enemy TNT.
 
 moderation.defuse.displayName = TNT Defuser
 
+# {0} = defuser player name
+# {1} = griefer player name
```

Comment:
`{1} = TnT owner player name`? Just not `griefer` imo

Reply:
Changed it to defusing and griefing player name